src/main/license/source-inclusions.yaml

```diff
@@ -40,3 +39,4 @@
 - id: js-yaml.js
 - id: jquery.form.js
 - id: marked.js
+- id: codemirror.js
```

Comment:
for LICENSE to generate correctly the detail for codemirror needs to be added to the extras file `brooklyn-dist`

we should tweak that generation to read from other projects.  although that can wait, when we use bower we'll be rewriting it anyway.